src/people/main/__tests__/ButtonDelete.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+import React from 'react';
+import { Button } from 'components/common';
+
+describe('Test for FocusedView Button', () => {
+  const OnClick = jest.fn();
+
+  it('enable delete button if there is no a hunter assigned or bounty is paid', () => {
```

Comment:
shouldnt this be `enable delete button if there is no hunter AND bounty is NOT paid`

@ecurrencyhodler what do you think?